package/Dockerfile.suc

```diff
@@ -17,5 +17,6 @@ COPY system-agent-uninstall.sh /opt/rancher-system-agent-suc/system-agent-uninst
 COPY bin/rancher-system-agent /opt/rancher-system-agent-suc
 COPY package/suc/run.sh /opt/rancher-system-agent-suc/run.sh
 COPY --from=kubectl /usr/bin/kubectl /usr/bin/
+RUN chmod +x /opt/rancher-system-agent-suc/rancher-system-agent
```

Comment:
I'm assuming this is something to do with GHA not keeping the executable flag on the produced binary..?

Reply:
Yes, you are right.

Reply:
see https://github.com/actions/download-artifact#limitations